plugins/UM3NetworkPrinting/src/Cloud/CloudOutputDevice.py

```diff
@@ -340,6 +342,37 @@ def openPrintJobControlPanel(self) -> None:
     def openPrinterControlPanel(self) -> None:
         QDesktopServices.openUrl(QUrl(self.clusterCloudUrl + "?utm_source=cura&utm_medium=software&utm_campaign=monitor-manage-printer"))
 
+    permissionsChanged = pyqtSignal()
+
+    @pyqtProperty(bool, notify = permissionsChanged)
+    def canReadPrintJobs(self) -> bool:
+        """
+        Whether this user can read the list of print jobs and their properties.
+        """
+        return "digital-factory.print-job.read" in self._account.permissions
```

Comment:
What is the difference between `digital-factory.print-job.read` and `cura.printjob.read` see https://docs.api.ultimaker.com/authentication/scopes.html does one superseeds the other? Are we still using the `cura.printjob.read` scope with this new permission scheme

Reply:
The OAuth scope system and the account permissions system are two different systems that work on top of each other in the cloud. We need both. We can assume that the OAuth scope can only be given if the account has that permission, and we always ask for this OAuth scope upon logging in, so it's sufficient to read if the permission is in the account.